Makefile

```diff
@@ -154,11 +154,17 @@ bin/%/submariner-operator: $(VENDOR_MODULES) main.go $(EMBEDDED_YAMLS)
 
 ci: $(EMBEDDED_YAMLS) golangci-lint markdownlint unit build images
 
-# Operator CRDs
+# Download controller-gen locally if not already downloaded.
+CONTROLLER_TOOLS_VERSION := 0.9.2
 $(CONTROLLER_GEN): $(VENDOR_MODULES)
 	mkdir -p $(@D)
-	$(GO) build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
+	test -s $(CURDIR)/bin/controller-gen || $(GO) build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
+	## TODO (Jaanki) Use go install instead
+	# $(GO) install sigs.k8s.io/controller-tools/cmd/controller-gen@v$(CONTROLLER_TOOLS_VERSION)
```

Comment:
Will this TODO be resolved in this PR? If not, what's preventing us from using `go install`?

Reply:
Using `go install` fails with `go: [sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2](http://sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2): cannot query module due to -mod=vendor` error. I am not sure whats happening here. This is a non-blocker and should not block this PR.